src/server/game/Chat/ChatCommands/ChatCommand.cpp

```diff
@@ -322,8 +322,14 @@ namespace Acore::Impl::ChatCommands
         oldTail = newTail;
     }
 
+    /* if we matched a command at some point, invoke it */
     if (cmd)
-    { /* if we matched a command at some point, invoke it */
+    {
+        if (!sScriptMgr->CanExecuteCommand(handler, cmdStr))
+        {
```

Comment:
You can remove the braces

Reply:
> You can remove the braces

Alright, it looks like the AzerothCore code style changed lol. I'll update it